src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -65,10 +63,7 @@ const wrapperClasses = prefixUtilityClasses([
   'align-items--flex-start',
 ]);
 
-const editButtonClasses = [
-  'usa-button-secondary',
-  ...prefixUtilityClasses(['margin--0', 'margin-top--1p5']),
```

Comment:
You can leave this as is for now, but I think this change will introduce a little extra space below the buttons and the bottom edge of "table cell" the contact info field lives in. But we can take another pass at getting the CSS totally nailed down when this is on staging.